contracts/Insurance.sol

```diff
@@ -32,17 +34,219 @@ contract Insurance is IInsurance {
     // Target percent of leveraged notional value in the market for the insurance pool to meet; 1% by default
     uint256 private constant INSURANCE_POOL_TARGET_PERCENT = 1e16;
 
+    uint256 public delayedWithdrawalCounter;
+    // Delayed withdrawal ID => DelayedWithdrawal
+    mapping(uint256 => LibInsurance.DelayedWithdrawal) delayedWithdrawalAccess;
+    // Address => Current delayedWithdrawal ID
+    mapping(address => uint256) public override accountsDelayedWithdrawal;
+    uint256 public override totalPendingCollateralWithdrawals;
+    // After 5 days, withdrawal can be executed for a 5 day window period
+    uint256 public constant delayedWithdrawalLock = 5 days;
+    uint256 public constant delayedWithdrawalWindow = 5 days;
+    StructuredLinkedList.List list;
+
+    // Approximate average gas it costs for each iteration in `scanDelayedWithdrawals(...)`
+    // AVERAGE_SCAN_GAS_PER_ACCOUNT = 41120;
+    // Approximate gas it costs to call `commitToDelayedWithdrawal(...)`
+    // AVERAGE_COMMIT_GAS = 384436;
+    // Approximate gas it costs to call `executeDelayedWithdrawal()`
+    // AVERAGE_EXECUTE_GAS = 132412;
+    // COMMIT_EXECUTE_GAS_SUM = AVERAGE_COMMIT_GAS + AVERAGE_EXECUTE_GAS;
+    // When committing, or executing a delayed withdrawal, spend half the amount of gas it would
+    // cost to run the function on scanning through the delayed withdrawals and deleting any expired ones
+    // SCAN_EXPIRED_WITHDRAWAL_COUNT = COMMIT_EXECUTE_GAS_SUM / 2 / AVERAGE_SCAN_GAS_PER_ACCOUNT;
+    // This ends up being 6 iterations. i.e. At most do 6 iterations through the pending delayed withdrawals
+    uint256 SCAN_EXPIRED_WITHDRAWAL_COUNT = 6;
```

Comment:
What does the system do in the case there are several expired withdrawals that are not updated? E.g 100 expired / executed withdrawal commits exist, does this mean that the insurance figures (getPoolHoldingsWithPending) are out of sync until 17 (100/6) new commits are created? Is this acceptable in the system, or is the owner required to manually scan withdrawals to ensure the figures are up to date.